ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -375,13 +372,6 @@ export default class ExperimentalTab extends PureComponent {
     ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
     toggle = this.keyringSnapsToggle();
```

Comment:
The default value here is null. we will only reassign this value to keyringSnapsToggle in the `keyring-snaps` build.